README.md

```diff
@@ -634,6 +634,7 @@ Sample result:
 
 ```js
 {
+  last_modified: 1456183930780,
```

Comment:
Note: if we want to use the same vocabulary as the server, then this should be `timestamp` and should be a string (`ETag` value)